web/assets/js/views/WorkQueueConsents.js

```diff
@@ -119,16 +246,52 @@ $(document).ready(function() {
 
     // Display custom error message
     $.fn.dataTable.ext.errMode = 'none';
-    $('#workqueue_consents').on('error.dt', function(e) {
+    $('#workqueue_consents').on('error.dt', function (e) {
         alert('An error occurred please reload the page and try again');
     });
 
     // Scroll to top when performing pagination
-    $('#workqueue_consents').on('page.dt', function() {
+    $('#workqueue_consents').on('page.dt', function () {
         //Took reference from https://stackoverflow.com/a/21627503
         $('html').animate({
             scrollTop: $('#filters').offset().top
         }, 'slow');
         $('thead tr th:first-child').trigger('focus').trigger('blur');
     });
+
+    $('.toggle-vis').on('click', function () {
+        var column = workQueueTable.column($(this).attr('data-column'));
+        column.visible(!column.visible());
+        var columnName = $(this).attr('name');
+        // Set column names in session
+        $.get("/s/workqueue/consent/columns", {columnName: columnName, checked: $(this).prop('checked')});
```

Comment:
Can we pass this route in from the server-side so that we can use the named route? Maybe as a data attribute like you did for the column definitions?